ui/config.yaml

```diff
@@ -1,18 +1,19 @@
 loglevel: DEBUG
 job:
+  base_path: "../examples/MLCommons"
```

Comment:
base path needs to be renamed to input path

Reply:
Done